DXRMapFixups/DeusEx/Classes/DXRFixupVandenberg.uc

```diff
@@ -704,19 +728,43 @@ function TimerMapFixes()
         break;
 
     case "14_Oceanlab_silo":
-        prevMapsDone = dxr.flagbase.GetBool('Heliosborn') &&  //Finished Vandenberg, mission 12
-            dxr.flagbase.GetBool('schematic_downloaded'); //Finished Ocean Lab, mission 14,
-        prevMapsDone = prevMapsDone || !#defined(injections) || dxr.flags.settings.goals<=0;
+        // by design, no infolink plays after killing Howard Strong if the missile hasn't been redirected
+        if (dxr.flagbase.GetBool('DXR_SiloEscapeHelicopterUnhidden')) {
+            if (dxr.flagbase.GetBool('DL_Savage3_Played')) {
+                // both goals completed, computer infolink played
+                player().StartDataLinkTransmission("DL_Dead");
+            } else {
+                // both goals completed, computer infolink not played
+                for (ce = GetConversation('DL_Savage3').eventList; ce != None; ce = ce.nextEvent) {
+                    if (
+                        ce.nextEvent != None &&
+                        ce.nextEvent.eventType == ET_Speech &&
+                        ConEventSpeech(ce.nextEvent).conSpeech.speech == "JC, go to the silo.  Make sure no one interferes with this launch."
+                    ) {
+                        ce.nextEvent = None;
```

Comment:
should only do this if goals rando is enabled `if(dxr.flags.settings.goals > 0)`